webextension/backgroundscript.js

```diff
@@ -36,6 +36,6 @@ genericBrowser.browserAction.onClicked.addListener(handleClick);
 
 function handleClick(event) {
     genericBrowser.tabs.create({
-        url: genericBrowser.runtime.getURL('/index.html')
+        url: genericBrowser.runtime.getURL('/www/index.html')
```

Comment:
I'm curious: is this change a necessary part of packaging the scripts? If not, perhaps it'd be worth separating into a separate commit and pull request?